commands/settings/index.js

```diff
@@ -0,0 +1,31 @@
+'use strict'
+
+const cli = require('heroku-cli-util')
+const co = require('co')
+
+function * run (context, heroku) {
+  const host = require('../../lib/host')
+  const util = require('../../lib/util')
+  const fetcher = require('../../lib/fetcher')(heroku)
+  const {app, args} = context
+
+  const db = yield fetcher.addon(app, args.database)
+
+  if (util.starterPlan(db)) throw new Error('This operation is not supported by Hobby tier databases.')
+  if (util.legacyPlan(db)) throw new Error('This operation is not supported by Legacy-tier databases.')
```

Comment:
Also either both "Hobby tier" and "Legacy-tier" should be hyphenated, or neither should be. And for what it's worth, I'm fine shipping after making that consistent, and then sorting the tier thing out later if necessary.